README.md

```diff
@@ -1 +1,214 @@
+  </ul>
+</details>
+
+<details>
+<summary>ES6 JavaScript</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript">ES6 Javascript</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Webpack</summary>
+  <ul>
+    <li><a href="https://webpack.js.org/guides/">Webpack</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+- **[HTML to prepare the page semantics]**
+- **[CSS to style the page]**
+- **[linters to improve code readability and spot syntax errors]**
+- **[is a static module bundler for modern JavaScript applications]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+
+### Prerequisites
+
+In order to run this project you need:
+
+- git installed.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```
+  cd To-do-list
+  git clone https://github.com/Obimbo07/To-do-list
+```
+--->
+
+### Install
+
+Install this project with:
+
+
+```sh
+  cd To-do-list
+  npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+
+```
+  live server
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+
+
+```npx hint .
+```
+
+### Deployment
+
+ Here is a link to the live demo of the website [Click here](https://obimbo07.github.io/To-do-list/) 
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Author1**
+
+- GitHub: [@Obimbo07](https://github.com/Obimbo07)
+- Twitter: [@Austine](https://twitter.com/austsine)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/austin-obimbo-9a613623a/)
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Implement a time section to specify task duration]**
+- [ ] **[A remainder to enable user get notified about task schedule]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Obimbo07/To-do-list/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+If you like this project , please show your support by:
+
+- Contributing to growth and improvement of our code base
+- Give us a 5 Star
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+Thank you github for the github pages
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
```

Comment:
- [OPTIONAL] Please remove the word `OPTIONAL` because you already have added the detail, so it is no more optional and it is just for guidance purposes for a developer while creating the `README`, but for normal person it is now part of your instructions.